src/store/store.js

```diff
@@ -2,82 +2,112 @@ import {makeAutoObservable} from 'mobx';
 
 class Store {
   BASE_URL = 'http://192.168.100.106:1339';
+
   categories = [];
+
   currentCategory = [];
+
   questionSets;
+
   chosenQuestionsSet = [];
+
   questions;
+
   currentQuestion = [];
+
   currentAnswers = [];
+
   chosenMode = '';
+
   showHelpDialog = false;
+
   isChooseModeDialog = false;
 
+  isAnswerHasChosen = false;
```

Comment:
not sure about this but as much as I remember you don't put empty lines between class variables